pkg/reconciler/autoscaling/kpa/resources/decider.go

```diff
@@ -60,6 +64,17 @@ func MakeDecider(ctx context.Context, pa *v1alpha1.PodAutoscaler, config *autosc
 	if x, ok := pa.TargetBC(); ok {
 		tbc = x
 	}
+
+	initScale := config.InitialScale
+	if revInitScale, ok := pa.Annotations[autoscaling.InitialScaleAnnotationKey]; ok {
+		revInitScaleInt, err := strconv.Atoi(revInitScale)
+		if err != nil {
+			logger.Errorf("Error processing revision initial scale %d: %v", revInitScale, err)
+		}
+		if revInitScaleInt > 1 {
```

Comment:
does this mean that if the global default is 0 I can't change it to 1 via annotation? or if it's 1 I can't change it to 0? is that the intent?